common/djangoapps/util/tests/test_markup.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for util.markup
+"""
+
+import unittest
+
+import ddt
+
+from edxmako.template import Template
+from util.markup import escape, HTML, ugettext as _, ungettext
```

Comment:
@cahrens Discussed with @dianakhuang and she added back escape() in ugettext() in order to get it to work, because it needs to result in a Markup object to work.  She also removed the "| h" from the usage comments.